rblxopencloud/__init__.py

```diff
@@ -25,7 +25,7 @@
 import requests
 
 VERSION: str = "2.0.0"
-VERSION_INFO: Literal['alpha', 'beta', 'final'] = "alpha"
+VERSION_INFO: Literal['alpha', 'beta', 'final'] = "beta"
```

Comment:
I'm not sure if we want to move into beta yet.

Reply:
This has been solved.